tests/_helpers/github-files.js

```diff
@@ -0,0 +1,53 @@
+const fs = require( 'fs' );
+const GitHubClient = require ( './github-client' );
+
+async function sendFiles( branch, files ) {
+	const nextFile = files.shift();
+
+	if( nextFile ) {
+		await sendFile( branch, nextFile.src, nextFile.dest );
+		return sendFiles( branch, files );
+	}
```

Comment:
What will it return if there is no `nextFile`?

Reply:
Have you considered using `for...of` loop instead of recursion here? Like:

```javascript
for ( const file of files ) {
  await sendFile( ... )
}
```

?